.travis.yml

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
please remove this blank line, and the blank line at the end of the file..

See them at https://github.com/andrewda/valid-github-username/pull/8/files

Reply:
Okay